server/api/user/index.js

```diff
@@ -1,7 +1,30 @@
 import express from 'express';
-
+import controller from './user.controller';
 const router = express.Router();
+// TODO: Verify user-id with token
+
+// GET => all user expenses for selected pool
+// Cannot GET /api/users/5/expense-pools-users/4/expenses
+router.get('/:id/expense-pools-users/:exp-pool-id/expenses',
+            controller.getUserPoolExpenses);
+
+// GET => all pools that a user is part of
+router.get('/:id/expense-pools-users', controller.getUserPools);
+
+// GET => all user expenses
+router.get('/:id/users-expenses', controller.getUserExpenses);
 
-// TODO: Fill out user api routes USER.
+// GET => info on one user
+router.get('/:id', controller.getUser);
+
+// POST => expense for user
```

Comment:
Unless / until we move this post into the ExpensePool router, line is changed to: `/:id/expense-pool/:expensePoolId` to reflect a user being added to a specific ExpensePool. The line above is now: `/:id/users-expenses/:expenseId` to reflect a specific expense being associated with one user.